ENYO-493: Duration Picker (hour, minute, second)

anishlg opened the pull request:
updated the old duration picker
https://github.com/enyojs/moonstone/pull/1978/

aarontam commented on source/DurationPicker.js:
```diff
@@ -0,0 +1,690 @@
+		*/
+
+		/* Interval Object */
+		var Interval = function (func, delay) {
```
Hi @anishlg, do you know why it was necessary to create all of this custom functionality? It appears to work fine, but I seem to recall we were originally utilizing `setInterval`.

anishlg replied:
Hi @aarontam, as i mentioned earlier the window.setInterval() function always had +/- 1 ms difference in each interval (1 s). By getting added this in every interval, this will considerably get wrong duration on long run. In current the moon.interval keep the difference consistent from the starting time by using {{enyo.perfNow}}.

aarontam replied:
Hi @anishlg, thanks for the clarification. I'm wondering if this shouldn't go into `enyo` core, maybe as part of `enyo.job` i.e. `enyo.repeatingJob`. It seems to be something that would be useful at a more general level, and could benefit from utilizing the job paradigm where a job can be named and a delay can be specified.

cc @Djspaceg @clinuz

anishlg commented:
Updated the Duration Picker
1. removed the setTimeout() function dependency and used the enyo core function enyo.startJob() to create the timer
2. removed the template property and used hour/minute/second hide/show properties

KunmyonChoi commented:
We closed the Jira because no app is planning to use this new control. Let's close the PR here.
